src/redmodding.ts

```diff
@@ -60,6 +85,146 @@ export const REDmoddingTools = [
   // REDmodDeploy,
 ];
 
+export const redModTool = (state: types.IState, gameId: string): types.IDiscoveredTool => {
```

Comment:
Minor, use the `Vortex*` types here and elsewhere

Reply:
Ah, I guess I missed that actually. I try to use the types from the vortex-wrapper generally, but there was so much I must have missed it lol

Reply:
I just added, no wonder you wouldn't remember, sorry 🤦🏻